src/com/lftechnology/java/training/srijan/basics/coreapp/controller/EmployeeManagementController.java

```diff
@@ -0,0 +1,186 @@
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.List;
+import java.util.Scanner;
+
+public class EmployeeManagementController {
+	private static final Logger LOGGER = Logger.getLogger(EmployeeManagementController.class.getName());
+	List<Employee> employees;
+	Employee employee;
+	Employee e = new Employee();
+	EmployeeServiceImpl employeeServiceImpl = ServiceFactory.getEmployeeService();
+
+	public EmployeeManagementController() {
+	}
+
+	public UserRole save(String userName, String password) {
+		e.setUserName(userName);
+		e.setPassword(password);
+		return employeeServiceImpl.save(e);
+	}
+
+	public void adminUserOptionHandler(int adminChoice) {
+		switch (adminChoice) {
+		case 1:
+			Employee employeeDetails = inputUserInfo(e);
+			employeeServiceImpl.addEmployee(employeeDetails);
+			break;
+		case 2:
+			employees = employeeServiceImpl.viewAllUser(e);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			LOGGER.log(Level.INFO, "Choose id of an Employee you want to Edit");
+			int employeeIdToEdit = editAndTerminateEmployee();
+			employee = employeeServiceImpl.findById(employeeIdToEdit);
+			LOGGER.log(Level.INFO, "{0}", employee);
+			Employee editEmployeeDetails = inputUserInfo(e);
+			employeeServiceImpl.updateEmployee(editEmployeeDetails);
+			break;
+		case 3:
+			LOGGER.log(Level.INFO, "Choose id of an Employee you want to terminate");
+			int terminateEmployee = editAndTerminateEmployee();
+			employeeServiceImpl.terminate(terminateEmployee);
+			break;
+		case 4:
+			employees = employeeServiceImpl.viewAllUser(e);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		default:
+			LOGGER.log(Level.INFO, "Wrong option, Default view functionality");
+			employees = employeeServiceImpl.viewAllUser(e);
+			LOGGER.log(Level.INFO, "{0}", employees);
+			break;
+		}
+	}
+
+	public Employee inputUserInfo(Employee e) {
+		Scanner input = null;
+		int choiceUserRole = 0;
+		int userStatus = 0;
+		String username = "";
+		String fullname = "";
+		String department = "";
+		String address = "";
+		String password = "";
+		try {
+			input = new Scanner(System.in);
+			LOGGER.log(Level.INFO, "Enter Username*:");
+			username = input.nextLine();
+			e.setUserName(username);
+			LOGGER.log(Level.INFO, "Enter Fullname*:");
+			fullname = input.nextLine();
+			e.setFullName(fullname);
+			LOGGER.log(Level.INFO, "Enter Department*:");
+			department = input.nextLine();
+			e.setDepartment(department);
+			LOGGER.log(Level.INFO, "Enter Address:");
+			address = input.nextLine();
+			e.setAddress(address);
+			LOGGER.log(Level.INFO, "Enter password*");
+			password = input.nextLine();
+			e.setPassword(password);
+			LOGGER.log(Level.INFO, "Enter User Role*:\n1.ADMIN\n2.NORMAL\n3.Default: NORMAL");
+			choiceUserRole = input.nextInt();
+			switch (choiceUserRole) {
+			case 1:
+				e.setUserRole(UserRole.ADMIN);
+				System.out.println(e.getUserRole());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)